compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/syntax/declarations/FunctionDefinitionTest.java

```diff
@@ -157,11 +157,6 @@ public void testFunctionWithInvalidExpressionStatement() {
         testFile("func-definition/func_def_source_24.bal", "func-definition/func_def_assert_24.json");
     }
 
-    @Test
-    public void testMissingFunctionKeywordWithQualifiers() {
-        testFile("func-definition/func_def_source_25.bal", "func-definition/func_def_assert_25.json");
-    }
-
```

Comment:
Any reason for removing these test cases?

Reply:
Now that parsing is independent of qualifiers
```bal
public isolated transcational foo(int i) returns boolean { }
```
is equivalent to
```bal
public foo(int i) returns boolean { }
```
Removed the test cases(5 tests) which were previously recovered based on the preceding qualifiers with a875674

Reply:
Anyway consider following code snippet.
```bal
public foo() returns boolean { }

public foo(int i) returns boolean { }
```

Recovery is
```bal
public  MISSING[function]foo() returns boolean { }

 INVALID[public] foo MISSING[error] MISSING[]( INVALID[int] i)  MISSING[;] MISSING[object] INVALID[returns]  INVALID[boolean] { } MISSING[] MISSING[;]
````
in the current master.

It seems something is wrong with the recovery of functions with parameters

Reply:
* https://github.com/ballerina-platform/ballerina-lang/blob/70c5342829156da8525211726b1548d2ad018271/compiler/ballerina-parser/src/test/resources/types/func-type/func_type_source_10.bal
* https://github.com/ballerina-platform/ballerina-lang/blob/70c5342829156da8525211726b1548d2ad018271/compiler/ballerina-parser/src/test/resources/types/func-type/func_type_source_12.bal

Consider above two test cases. They both same only different is qualifiers.  Also there are invalid tokens in between qualifiers in lot of places and now some of them match with the switch cases in the parsing methods and parser going haywire. Since those test cases don't  test any granular level recovery we don't need to keep them right?

* https://github.com/ballerina-platform/ballerina-lang/blob/70c5342829156da8525211726b1548d2ad018271/compiler/ballerina-parser/src/test/resources/declarations/isolated-object-methods/isolated_object_method_source_02.bal
* https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-parser/src/test/resources/declarations/func-definition/func_def_source_25.bal

Above two test cases the same and they are broken due to #26460. I think adding one test case when fixing that issue would be sufficient.